ci/Jenkinsfile.groovy

```diff
@@ -692,9 +692,9 @@ ansiColor('xterm') {
                             unstash 'ppc64le_centos7-py35-whl'
                             unstash 'sdist-tar'
                         }
-                        docker.withRegistry("https://docker.h2o.ai", "docker.h2o.ai") {
+                        docker.withRegistry("https://harbor.h2o.ai", "harbor.h2o.ai") {
                             withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: "awsArtifactsUploader"]]) {
-                                docker.image("docker.h2o.ai/s3cmd").inside {
+                                docker.image("harbor.h2o.ai/s3cmd").inside {
```

Comment:
For this I believe it is harbor.h2o.ai/library/s3cmd no?

Reply:
Thanks, should be corrected.